src/solvers/first_order_unsteady_solver.C

```diff
@@ -131,4 +137,92 @@ bool FirstOrderUnsteadySolver::compute_second_order_eqns(bool compute_jacobian,
   return compute_jacobian;
 }
 
+
+void FirstOrderUnsteadySolver::add_dot_var_dirichlet_bcs( unsigned int var_idx,
+                                                          unsigned int dot_var_idx )
```

Comment:
Okay, *this* one is actually broken.  We can't set u_dot = f on a boundary where u = f, except in the f = 0 case.  In general we need u_dot = df/dt... and I'm not sure what the right way to get that is.